tools/pelela-cli/templates/base-template-for-cli/public/help.html

```
@@ -0,0 +1,439 @@
+          </tr>
+          <tr>
+            <td><span class="attribute">bind-style</span></td>
+            <td>Any element</td>
+            <td>Bind element's style object to a view model property</td>
+            <td><span class="code">&lt;div bind-style="dynamicStyle" /&gt;</span></td>
+          </tr>
+          <tr>
+            <td><span class="attribute">bind-enabled</span></td>
+            <td><span class="code">&lt;input&gt;</span>, <span class="code">&lt;select&gt;</span>, <span class="code">&lt;button&gt;</span>, <span class="code">&lt;textarea&gt;</span>, <span class="code">&lt;optgroup&gt;</span>, <span class="code">&lt;option&gt;</span>, <span class="code">&lt;fieldset&gt;</span></td>
+            <td>Bind element's enabled state to a view model property</td>
+            <td><span class="code">&lt;button bind-enabled="canSubmit" /&gt;</td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </span> ], start tag match failed [ <span class="code"> ] on line 358.

(tag-pair)